server/JsDbg.Gdb/GdbDebugger.cs

```diff
@@ -8,6 +8,8 @@ namespace JsDbg.Gdb {
     class GdbDebugger : IDebugger {
 
         public GdbDebugger() {
+            // TODO: Initialize from LookupTypeSize("void*")

```

Comment:
Can we get this TODO fixed with (or before) this change? I think we should keep 32-bit and 64-bit working equally well at all times.

Reply:
Hm, it does look like nobody called IsPointer64Bit before. So the value was wrong but ignored... OK, I will fix that tomorrow.

Reply:
OK, I updated this pull request to initialize IsPointer64Bit correctly. Please take another look!